src/main/java/visualizations/ScatterChart.java

```diff
@@ -0,0 +1,77 @@
+package visualizations;
+
+
+import datastructs.NumericSample;
+import tech.tablesaw.api.DoubleColumn;
+import tech.tablesaw.api.Table;
+import tech.tablesaw.plotly.Plot;
+import tech.tablesaw.plotly.api.BubblePlot;
+import tech.tablesaw.plotly.api.ScatterPlot;
+
+/**
+ * Class that plots basic Scatter Charts.
+ */
+
+public class ScatterChart {
+
+    public class ScatterChartOptions {
+        public String chartTitle;
+        public String xAxisName;
+        public String yAxisName;
+        public String groupColName;
+        public String sizeColName;
+    }
+
+    /**
+     * Plots a scatter chart given the chart title,
+     * data set in Table format,
+     * and two numeric columns
+     * created from a numeric sample
+     * of the data set.
+     */
+
+    public static void plotScatter(NumericSample x, NumericSample y, ScatterChartOptions options){
```

Comment:
The rest of the functions in this class pass first the options and then the data. Why did you decide that this one is different?

Reply:
reversed the order & updated documentation.